ui/Screen/Settings.svelte

```diff
@@ -249,11 +248,11 @@
       </div>
       <div class="section-item">
         <div class="info">
-          <p class="typo-text-bold">Join the developer chat</p>
+          <p class="typo-text-bold">Join the community chat</p>
         </div>
         <div class="action">
-          <a class="typo-link" href="irc://freenode:1/radicle">
```

Comment:
I guess we can then remove this now:
https://github.com/radicle-dev/radicle-upstream/blob/7977a5dcf4a05eb19e395049f213e9f5b1659d80/native/main.ts#L139